requirements.txt

```diff
@@ -2,4 +2,5 @@ robotframework
 robotframework-seleniumlibrary==5.1.3
 robotframework-pabot
 browserstack-local
-selenium==3.141.0
+selenium
```

Comment:
Pin to version 4?

Reply:
by default it will install sel4 but if the user has sel3 installed no reason to upgrade that to 4 imo.. as sdk works with both